quandl/bulkdownload.py

```diff
@@ -0,0 +1,26 @@
+from quandl.errors.quandl_error import InvalidRequestError
+from .utils.api_key_util import ApiKeyUtil
+from .model.database import Database
+
+
+def bulkdownload(database, **kwargs):
+    """Downloads an entire database.
+    :param str database: The database code to download
+    :param str filename: The filename for the download. \
+    If not specified, will download to the current working directory
+    :param str api_key: Most databases require api_key for bulk download
+    :param str download_type: 'partial' or 'complete'. \
+    See: https://www.quandl.com/docs/api#database-metadata
+    """
+
+    # discourage users from using authtoken
+    if 'authtoken' in kwargs:
+        error_msg = "Parameter authtoken is no longer supported. \
```

Comment:
could you put all user facing messages into this https://quandl.atlassian.net/browse/AP-1664? so these messages can be finalized by someone.